lib/vets/attributes/value.rb

```diff
@@ -42,6 +42,14 @@ def cast_boolean(value)
       def coerce_to_class(value)
         return value if value.is_a?(@klass) || value.nil?
 
+        if @klass == DateTime
```

Comment:
Would we want to support Integer here (epoch)?

```ruby
Time.at(value).to_datetime if value.is_a? Integer
```

Reply:
That's an interesting idea. I don't see why we can't have that